homeassistant/components/microsoft_face/__init__.py

```diff
@@ -313,8 +319,8 @@ async def call_api(self, method, function, data=None, binary=False, params=None)
 
         try:
             async with asyncio.timeout(self.timeout):
-                response = await getattr(self.websession, method)(
-                    url, data=payload, headers=headers, params=params
+                response = await self.websession.request(
+                    method, url, data=payload, headers=headers, params=params
```

Comment:
I couldn't see any reasy why it was previously using `getattr`, so I adjusted it to improve code quality